tests/test_functional.py

```diff
@@ -19,6 +20,29 @@
 from bs4 import BeautifulSoup
 
 
+def is_valid_orca_nc_url(url_id, resp):
+    orca_base = os.getenv("ORCA_ROOT")
+    with session_scope(os.getenv("DSN")) as sesh:
+        q = sesh.query(DataFile.filename).filter(DataFile.filename.contains(url_id))
+    storage_path = q.one()[0].strip("/")
+
+    assert resp.status == "301 Moved Permanently"
+    assert resp.content_type == "text/plain"
+    assert all(dim in resp.location for dim in ["time", "lat", "lon"])
+    assert orca_base in resp.location
+    assert storage_path in resp.location
+    return True
```

Comment:
Minor caveat: This function is used as an assertion -- `assert is_valid_orca_nc_url(...)` -- but itself also does assertions. A mix that takes a little thought to understand. We could make it uniform in two ways:
- Remove the final `assert True` here name the function assert_is_valid_orca_nc_url`. Preferred, I think.
- Return `False` for each failing condition asserted above. This loses the specificity you get when it fails, though.